haydn.json

```diff
@@ -4,12 +4,12 @@
 			"maintainer": "Vinayak",
 			"oem": "Xiaomi",
 			"device": "Mi 11X Pro",
-			"filename": "crDroidAndroid-14.0-20231115-haydn-v10.0-BETA.zip",
-			"download": "https://sourceforge.net/projects/crdroid/files/haydn/10.x/crDroidAndroid-14.0-20231115-haydn-v10.0-BETA.zip/download",
-			"timestamp": 1700015378,
-			"md5": "3d6ab3a49d4622d2be54afacd700aab3",
-			"sha256": "78cf9abd18fed30a2f6cc42f04ea82139c1ec5f84c136eba4dec6bd26777919d",
-			"size": 1758312963,
+			"filename": "crDroidAndroid-14.0-20231220-haydn-v10.0-BETA.zip",
```

Comment:
filename hints to build from 20.12 while commit message from 23.12
please update build to proper stable source code